packages/mask/src/extension/popups/hooks/index.ts

```diff
@@ -0,0 +1,15 @@
+export * from './useContactsContext.js'
```

Comment:
This way will introduce side effects unintentionally. Just rely on IDE auto importing.